sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Utils.java

```diff
@@ -94,6 +100,18 @@ public static String encodeBase64String(byte[] binaryData) {
         return encodedString;
     }
 
+    public static String decodeAsUTF8String(String inputString) {
+        if (inputString == null || inputString.isEmpty()) {
```

Comment:
Can we use StringUtils.isEmpty here ?

Reply:
Yes, we can do that too.

Since this is implementation detail, will change it in next PR .